src/content/docs/aspen.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Aspen
+description: Check out Aspen, the privacy focused HTTP client.
+---
+
+Aspen is a Treblle's free, native macOS application designed specifically for testing REST APIs. Built with privacy in mind, it requires no login and stores no data, ensuring a smooth, local testing experience.
+
+Aspen comes with an integrated [AI assistant](https://treblle.com/product/alfred) to help automate API integration tasks, such as generating data models, Open API Specifications (OAS), and integration code.
+
+## Getting Started
+
+1. **Download Aspen**  
+    You can download Aspen from the [Apple Mac Store](https://apps.apple.com/us/app/aspen-your-apis-best-friend/id6475777033).
+
+   <img
+     src="/platform/aspen/download.png"
+     width="800"
+     style={{ borderRadius: "8px" }}
+   />
+
+2. **Install Aspen**  
+   After downloading, install the app on your Mac and open it.
+
+   <img
+     src="/platform/aspen/aspen.png"
```

Comment:
This image is not the correct view of Aspen when someone installs the app for the first time. I recommend you save your collections somewhere, delete everything and take a new screenshot.

Reply:
Added a new image